tests/estimators/object_detection/test_pytorch_yolo.py

```diff
@@ -367,7 +367,7 @@ def test_compute_loss(art_warning, get_pytorch_yolo):
         # Compute loss
         loss = object_detector.compute_loss(x=x_test, y=y_test)
 
-        assert pytest.approx(11.20741, abs=0.9) == float(loss)
+        assert pytest.approx(11.20741, abs=1.5) == float(loss)
```

Comment:
YOLO generates inconsistent loss for the same input. Fix is to freeze batch_norm and drop_out layers. However freezing these layers results in compute_loss returning NaN for some inputs due to a known issue in the python YOLO library. See #2148 for detail.